.github/CODE_OF_CONDUCT.md

```diff
@@ -8,19 +8,19 @@ In the interest of fostering an open and welcoming environment, we as contributo
 
 Examples of behavior that contributes to creating a positive environment include:
 
-* Using welcoming and inclusive language
```

Comment:
this is a good opportunity for us to centralize the CoC. let's nuke this doc and refer folks to Rollup's main project CoC

```md
This repository has adopted and follows the primary Rollup Code of Conduct https://github.com/rollup/rollup/blob/master/CODE_OF_CONDUCT.md
```